language/functional_tests/tests/testsuite/generics/resource_safety/borrow_global.mvir

```diff
@@ -1,18 +0,0 @@
-module M {
-    resource Foo<T> { x: T }
```

Comment:
Can we keep this test, but do `resource Foo<T>` -> `struct Foo<T>` (and similar for the tests below)?